.spi.yml

```diff
@@ -0,0 +1,22 @@
+version: 1
+external_links:
+  documentation: 'https://aws.amazon.com/sdk-for-swift'
+builder:
+  configs:
+  - platform: macosXcodebuild
+    target: AWSSTS
```

Comment:
Those were the two I was trying to avoid because they're so big, lol.  They take a very long time to compile, especially the `Models.swift` in each of those.

I don't think it shows anywhere except in the config file and in the build logs which specific target we build, so it makes no difference to someone looking at our listing.

Reply:
gotcha, makes sense then!